### DISCUSSION TITLE
Using the create3 examples for slam, how can I get the lidar motor to shut off?

### DISCUSSION THREAD

**Discussion by bdawkinsroj** [iRobotEducation/create3_docs#596] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which version of ROS 2 is installed on your computer?

None

### Which firmware version is installed on your robot?

latest humble release

### Which RMW is your robot running?

I don't know

### Are there multiple Create 3 robots connected to your network?

None

### Is multicast enabled?

None

### What is the Adapter Board's USB/BLE Toggle currently switched to?

None

### Describe your question.

How am I able to remotely turn off the lidar when I'm not using it. When using it remotely it'd be convenient to not have to unplug it every time.

**Comment by shamlian** [comment]
Hi there,

The Create 3 robot does not ship with a LIDAR. Perhaps you have a derivative product, like Clearpath's Turtlebot 4? If so, you might find an answer [here](https://github.com/turtlebot/turtlebot4/issues/).

**Comment by CFBerryhill** [comment]
A lot of LiDARS just spin by default if they're plugged in - for example, the RPLidar! My  solution is to leave it unplugged unless im currently using it.

**Comment by slowrunner** [comment]
> How am I able to remotely turn off the lidar when I'm not using it

First you need to review the API for your particular lidar.  There is likely a lidar.stop() function that can be called by your software.
BUT, second you have to modify your particular ROS2 lidar driver to stop publishing the /scan topic when the lidar is stopped.  If you don't do this, the ROS2 lidar driver will likely throw an exception or at the very least send invalid scan topics.

As others have stated - the Create3 does not ship with a LIDAR, so perhaps ask on the forum that supports your LIDAR.
